flavor.json

```diff
@@ -50,57 +50,76 @@
         {
           "seconds": 4,
           "text": [
-            "Welcome to the game",
+            "Velcom all, and thanks vor coming to my Vampire Party",
             "Salutations and welcome to Vampire Party",
-            "Thanks for joining the party"
+            "Thanks for joining the party",
+            "So glad you could make it. You won't live....to regret it"
           ]
         },
         {
-          "seconds": 3,
-          "text": "This is some text"
+          "seconds": 9,
+          "text": "Here in our gloomy town of Spoons we seem to have run into a bit of a vampire problem. Maybe because of our unforunate positioning ontop of a gaping..HellNose. Who is to really know?"
```

Comment:
Spelling:
*`unfortunate`
*`on top`?  my spell checker is giving error for ontop
*Did you mean to write `HellNose`.   In buffy they say `Hellmouth` so I think `Hellnose` would make more sense